cantera/build_devel.bat

```diff
@@ -4,6 +4,20 @@ ECHO ************************
 
 CALL scons install
 
+RD /S /Q "%PREFIX%/share/man"
+
+REM TODO: update Makefile and SConstruct files (if applicable), see issue #37
+REM Currently, `conda-build` isn't able to patch Makefiles properly
+REM Instead, just copy *.cpp sources
+RD /S /Q "%PREFIX%/share/cantera/samples"
+ROBOCOPY "%SRC_DIR%/samples/cxx" "%PREFIX%/share/cantera/samples/cxx" /S /E
+DEL /F "%PREFIX%/share/cantera/samples/cxx/SConscipt"
+DEL /F "%PREFIX%/share/cantera/samples/cxx/SConstruct.in"
+DEL /F "%PREFIX%/share/cantera/samples/cxx/Makefile.in"
```

Comment:
This is probably an upstream issue, but the `.in` files shouldn't be installed at all, SCons should be filling in the templates.

Reply:
In this case, I am copying from `%SRC_DIR%`, which do have the templates. Hopefully, with #37 fixed, all of this will go away ...

Reply:
Ah, I missed that. I don't think you should copy from the SRC_DIR, it should be from the installed location, right? And it shouldn't require any copying, if everything is installed to the right place.

Reply:
This is on purpose, as the build process creates makefiles in individual sub folders, which are difficult to scrub. I don’t see what speaks against copying from the source in this case as a temporary fix for #37?

Reply:
Honestly, the only alternative I see here is to not remove the files at all. The files contents are dubious, but we do consider them being ‘experimental’.